### DISCUSSION TITLE
How to use Web Browsing Capability?

### DISCUSSION THREAD

**Discussion by bien-phillip** [open-webui/open-webui#905] [General]
Hi community!

I can find Web Browsing Capability from README. However, I don't know how to use this feature. I try to find use case from document and google but I can't find any informaiton.

Please how to use Web Browsing Capability.

Thanks.

**Comment by justinh-rahb** [comment]
First type **#** followed by a URL and then click on it when it pops up above the text box:
<img width="760" alt="Screenshot 2024-02-19 at 8 34 23 PM" src="https://github.com/open-webui/open-webui/assets/52832301/233a14e6-a39e-4faf-bbd5-681fdcace901">

Once you've done that, the web page is now indexed into RAG and you can ask a question about it:
<img width="760" alt="Screenshot 2024-02-19 at 8 34 29 PM" src="https://github.com/open-webui/open-webui/assets/52832301/b798d6d8-5626-42eb-9de4-4715a62d3d67">

**Comment by m-rio** [comment]
does this index the entire website? ie. example.com, example.com/section1, example.com/section2, etc?

**Reply by Printaga** [reply]
No, only that specific URL.

**Reply by alkeryn** [reply]
would be neat if it could.
also would be nice if we could add a link in documents.

**Comment by mulkproject** [comment]
The challenge is to be able to by pass CAPTCH, POP UP modals, Authentication.

**Reply by justinh-rahb** [reply]
Pipeline server: https://github.com/open-webui/pipelines

**Comment by freund-masasana** [comment]
Is there a way to use addresses without http? If I type in "#www.example.com" nothing happens. Does the address have to have http(s) at the start and if so, then why? While I do understand the use case when pasting complete URLs, I don't think users will always go and copy paste a page they are already on, right? And if it is currently not supported, what would it take to support it?
